docs/versioned/install/installing-gateway-api.md

```diff
@@ -0,0 +1,64 @@
+---
+audience: administrator
+components:
+  - serving
+function: how-to
+---
+
+# Installing Gateway API for Knative
+
+This page shows how to install the Kubernetes Gateway API. You can install either the standard or experimental Gateway API channel.
+
+## Before you begin
+
+This installation requires the following prerequisites:
+
+- A Kubernetes cluster with the Knative Serving component installed.
+- A Gateway Controller. You can install a Gateway Controller by using one of the following guides:
```

Comment:
We should link to the Gateway API top-level guide, rather than duplicating all of the sub-guides.  I think https://gateway-api.sigs.k8s.io/implementations/ is probably the best page for linking to the options (yes, there are more than a dozen GA implementations).